### DISCUSSION TITLE
nvidia/l4t-base image incompatible with linux/amd64 platform

### DISCUSSION THREAD

**Discussion by palpen** [FamousDirector/DALLEePaperFrame#5] [General]
So I tried building the image in `Dockerfile.api` but keep getting the following error

```
[Warning] The requested image's platform (linux/arm64) does not match the detected host platform (linux/amd64) and no specific platform was requested
```

I've tried specifying the platform using `--platform` flag but with no luck. I'm running the `setup_server.sh` command on Ubuntu 20.04.03 LTS.

Any suggestions on a workaround?

**Comment by FamousDirector** [comment]
I see what happened. I've done some repo tidying now. Try after I do new PR. All of your mentioned issues should be fixed.

**Reply by palpen** [reply]
Thanks for fixing the issues. It works now :)
